perl6/lib/Cutie.pm

```diff
@@ -1,3 +1,3 @@
 role Cutie {
-  has Bool $.accept-hugs is rw = False;
+  method accept-hugs { … }
```

Comment:
I like this.

My instinct is to just expect it to eventually converge on APL :P